File: Medium/JavaScript/453. Minimum Moves to Equal Array Elements.js
var minMoves = function(nums) {
    let sum = 0;
    let min = nums[0];

    for(let i = 0; i < nums.length; i++){
        if(nums[i] < min)
            min = nums[i];
        sum += nums[i];
    }

    return sum - (nums.length * min);
};

File: Medium/Python/453. Minimum Moves to Equal Array Elements.py
class Solution(object):
    def minMoves(self, nums):
        my_sum = 0
        minimum = nums[0]

        for i in range(len(nums)):
            if nums[i] < minimum:
                minimum = nums[i]
            my_sum += nums[i]
        
        return my_sum - (len(nums) * minimum)
